src/screens/UserPortal/Events/Events.module.css

```diff
@@ -15,7 +15,7 @@
 }
 
 .colorLight {
-  background-color: #f1f3f6;
+  /* background-color: #f1f3f6; */
```

Comment:
```suggestion
 
```

nit: can be removed all commented code, if it's not used. Everything seems good except this small fix :)